core/src/test/resources/com/graphhopper/reader/osm/test-barriers.xml

```diff
@@ -4,16 +4,16 @@
         <tag k="name" v="Wiesbaden-Naurod"/>
     </node>
     <node id="20" lat="52" lon="9.4" version="1">
-        <tag k="barrier" v="gate"/>
```

Comment:
gate is passable by default so we either need a blockByDefaultBarrier or an additional access tag